scripts/babel/default-options.js

```diff
@@ -10,8 +10,19 @@
 'use strict';
 
 var babelPluginModules = require('./rewrite-modules');
+var babelPluginAutoImporter = require('./auto-importer');
 var inlineRequires = require('./inline-requires');
-var plugins = [babelPluginModules];
+
+var plugins = [
+  {
+    position: 'after',
+    transformer: babelPluginAutoImporter,
+  },
+  {
+    position: 'after',
+    transformer: babelPluginModules,
+  },
+];
```

Comment:
Ugh, I guess Relay does use this file so they would want it. Let's leave it and I'll ship it as a minor version.